integration-tests/hibernate-reactive-postgresql/src/main/java/io/quarkus/it/hibernate/reactive/postgresql/HibernateReactiveTestEndpoint.java

```diff
@@ -129,25 +116,23 @@
     @Produces(MediaType.APPLICATION_JSON)
     public Uni<String> reactiveUpdate() {
         final String NEW_NAME = "Tina";
-        return mutinySession
-                .flatMap(mutinySession -> {
-                    return populateDB()
-                            .flatMap(junk -> mutinySession.find(GuineaPig.class, 5))
-                            .map(pig -> {
-                                if (NEW_NAME.equals(pig.getName()))
-                                    throw new AssertionError("Pig already had name " + NEW_NAME);
-                                pig.setName(NEW_NAME);
-                                return pig;
-                            })
-                            .flatMap(junk -> mutinySession.flush())
-                            .flatMap(junk -> selectNameFromId(5));
-                });
+        return populateDB()
+                .then(() -> mutinySession.find(GuineaPig.class, 5))
+                .invoke(pig -> {
+                    if (NEW_NAME.equals(pig.getName()))
+                        throw new AssertionError("Pig already had name " + NEW_NAME);
+                    pig.setName(NEW_NAME);
+                })
+                .then(() -> mutinySession.flush())
+                .then(() -> selectNameFromId(5));
     }
 
     private Uni<RowSet<Row>> populateDB() {
-        return pgPool.query("DELETE FROM Pig").execute()
-                .and(pgPool.query("DELETE FROM Cow").execute())
-                .flatMap(junk -> pgPool.preparedQuery("INSERT INTO Pig (id, name) VALUES (5, 'Aloi')").execute());
+        return Uni.combine().all().unis(
```

Comment:
@cescoffier , @gavinking this line threw me off a bit; is there a better / shorter way of expressing this?

All I want is to ensure both tables are empty; I could have used a simple "then", but wanted to be fancy and take advantge of the fact that order of execution of these is not important.

I noticed the "and()" operator is deprecated, couldn't find a better way than this.